### PR TITLE
e2e: add missing CLUSTER_NAME

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: integration/project.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ test:
     - "VAULT_TOKEN=mytoken"
     - "CIRCLE_SHA1=${CIRCLE_SHA1}"
     - "CIRCLECI=${CIRCLECI}"
+    - "CLUSTER_NAME=${CLUSTER_NAME}"
```
Comment (xh3b4sd) [inline integration/project.yaml]: I do not think this is missing. The tests write it in the go code now. Why should we add it?

Comment (fgimenez) [reply integration/project.yaml]: We talked about this here https://github.com/giantswarm/aws-operator/pull/831#discussion_r173858509 we need only to set it when it is not already set.

Comment (xh3b4sd) [reply integration/project.yaml]: But in which case does this fail now? Locally?

Comment (fgimenez) [reply integration/project.yaml]: Yes, locally it tries to set the cluster name based on `CIRCLE_SHA1`, which might be not set or set to a value that doesn't match with the chart pushed to quay.

**Pull request merged**